app/routes/status.rb

```diff
@@ -0,0 +1,41 @@
+module Citygram::Routes
+  class Status < Grape::API
+    version 'v1', using: :header, vendor: 'citygram'
```

Comment:
Updated Status for engine light to be outside of the API and simply return json.

Request:
http://localhost:5000/status

Results:
{"status":"ok","dependencies":["postgres","twilio"],"updated":1438124453}